sdk/identity/identity/Troubleshooting.md

```diff
@@ -137,41 +166,79 @@ The tenant ID is the Global Unique Identifier (GUID) that identifies your organi
 
 ### Client secret credential issues
 
-#### Client secret argument
+#### `AuthenticationRequiredError`
```

Comment:
This seems to be the first title within code quotes in this document. Is it intentional?

Reply:
I think i just copied that from .NET style. I will remove it